pkg/auth/commands/login.go

```diff
@@ -114,13 +117,15 @@ func Login() *cobra.Command {
 	flags.StringVarP(&clientID, clientIDFlagName, "", "3afae9ff-48e6-45f3-b0e8-37658b7271b7", "oauth client id")
 	flags.StringVarP(&apiURL, apiURLFlagName, "", "https://api.apono.io", "apono api url")
 	flags.StringVarP(&appURL, appURLFlagName, "", "https://app.apono.io", "apono app url")
+	flags.StringVarP(&portalURL, portalURLFlagName, "", "https://portal.apono.io", "portal url")
```

Comment:
we need to expose the client api under api.apono.io, not under the portal.apono.io